server/src/instant/reactive/session.clj

```diff
@@ -479,7 +484,13 @@
                                :attributes {:worker-n n}})
          (do (grouped-queue/process-polling!
               receive-q
-              (fn [item] (straight-jacket-process-receive-q-item store-conn eph-store-atom n item)))
+              {:reserve-fn receive-worker-reserve-fn
+               :process-fn (fn [_ [{{:keys [op]} :item :as entry} :as batch]]
+                             (tracer/with-span! {:name "receive-worker/process-receive-q-item"
+                                                 :attributes {:work-n n
+                                                              :op op
```

Comment:
Might be nice to log the group-key here too.

Reply:
Done!